star-wars-browser/src/store/planets/reducer.js

```diff
@@ -0,0 +1,14 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { initialState } from './defaultState';
+import { setPlanets } from './actions';
+
+const planetsReducer = createReducer(initialState, (builder) => {
+  builder
+    // eslint-disable-next-line no-unused-vars
```

Comment:
to chyba możesz usunąć - używasz plantes i action

Reply:
poprawione